src/ra.erl

```diff
@@ -1132,13 +1133,16 @@ key_metrics({Name, N} = ServerId) when N == node() ->
                end,
     case whereis(Name) of
         undefined ->
-            Counters#{state => noproc};
+            Counters#{state => noproc,
+                      non_voter => noproc};
```

Comment:
I think this should be `unknown` rather than `noproc` - I need to think of some way to report the leader's view of voter status in the case where some members are not reachable. It doesn't affect this PR though.